packages/dd-trace/src/ci-visibility/exporters/git/git_metadata.js

```diff
@@ -35,9 +35,7 @@ function getCommonRequestOptions (url) {
       'dd-api-key': process.env.DATADOG_API_KEY || process.env.DD_API_KEY
     },
     timeout: 15000,
-    protocol: url.protocol,
-    hostname: url.hostname,
-    port: url.port
+    url
```

Comment:
Are you sure this will work with UDS and such ? there is a reason why we pass many parameters

Reply:
mmm I don't think we'll ever use UDS for this call, but as far as I can see, `packages/dd-trace/src/exporters/agent/writer.js` is using only `url` 🤔

Reply:
It would be safer to make sure that everything has the same set of functionalities. It's possible someone might be using UDS in CI.

Reply:
yeah I think it _should_ work the same way as it does for the agent writer.